cmd/open.go

```diff
@@ -0,0 +1,37 @@
+package cmd
+
+import (
+	"github.com/j178/leetgo/leetcode"
+	"github.com/pkg/browser"
+	"github.com/spf13/cobra"
+)
+
+var openCmd = &cobra.Command{
+	Use:   "open [qid]",
+	Short: "open a question in browser",
+	Example: `leetgo open
+leetgo open today
+leetgo open 549
+leetgo open two-sum`,
+	Args: cobra.MaximumNArgs(1),
```

Comment:
we can use `cobra.ExactArgs(1)` here, then remove the `if len(args) > 0 ` at line 21.